docs/src/pages/index.tsx

```diff
@@ -36,19 +36,17 @@ interface Props {
   location: WindowLocation;
   path: string;
 }
-
 function GatsbyLinkToButton({ href, ...props }: { href: string }) {
   return <Link to={href} {...props} />;
 }
-
 export default function Home({ location, path }: Props) {
   const [searchOpen, setSearchOpen] = React.useState<boolean>(false);
   return (
     <Layout
       location={location}
       title="Orbit design system"
       description="An open source design system for your next travel project."
-      path="/"
+      path={path}
```

Comment:
We're deliberately hardcoding the path. ATM I forgot why, but using `path` didn't work out well in some cases, so delete it from the destructuring statement instead.